src/payment/PaymentMethodSelect.jsx

```diff
@@ -24,15 +26,23 @@ function PaymentMethodSelect({ intl }) {
             alt={intl.formatMessage(messages['payment.page.method.type.credit'])}
           />
         </button>
-        <PayPalButton />
-        <ApplePayButtonContainer />
+
+        <PayPalButton
+          className={classNames({ 'skeleton-pulse': loading })}
```

Comment:
What does `skeleton-pulse` mean?

Reply:
It's a class name for making an element fade in and out – you can see the animation in the screenshot (obviously :P)